### PR TITLE
able to set min mag for subduction

### PR DESCRIPTION
**Pull request by chrisdicaprio**

### CODE REVIEW

File: src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_SubductionInversionConfiguration.java [hunk 1]
```diff
@@ -144,7 +145,7 @@ public static NZSHM22_SubductionInversionConfiguration forModel(InversionModels
 
 		// setup MFD constraints
 		NZSHM22_SubductionInversionTargetMFDs inversionMFDs =  new NZSHM22_SubductionInversionTargetMFDs(rupSet, totalRateM5, bValue, mfdTransitionMag,
-				mfdUncertaintyWeightedConstraintWt, mfdUncertaintyWeightedConstraintPower, mfdUncertaintyWeightedConstraintScalar);
+				mfdMinMag,mfdUncertaintyWeightedConstraintWt, mfdUncertaintyWeightedConstraintPower, mfdUncertaintyWeightedConstraintScalar);
```
Comment (voj) [inline src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_SubductionInversionConfiguration.java:148]: there should be a space after the comma

File: src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_SubductionInversionRunner.java [hunk 2]
```diff
@@ -55,7 +69,7 @@ protected NZSHM22_SubductionInversionRunner configure() throws DocumentException
 		NZSHM22_SubductionInversionConfiguration inversionConfiguration = NZSHM22_SubductionInversionConfiguration
 				.forModel(inversionModel, rupSet, initialSolution, mfdEqualityConstraintWt, mfdInequalityConstraintWt,
 						mfdUncertWtdConstraintWt, mfdUncertWtdConstraintPower, mfdUncertWtdConstraintScalar,
-						totalRateM5, bValue, mfdTransitionMag);
+						totalRateM5, bValue, mfdTransitionMag,mfdMinMag);
```
Comment (voj) [inline src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_SubductionInversionRunner.java:72]: space

File: src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_SubductionInversionTargetMFDs.java [hunk 3]
```diff
@@ -85,7 +87,7 @@ public NZSHM22_SubductionInversionTargetMFDs(NZSHM22_InversionFaultSystemRuptSet
 		}
 		
 		//Doctor the target, setting a small value instead of 0
-	 	totalTargetGR.setYofX((x, y) -> {return (x < MINIMIZE_RATE_BELOW_MAG) ? MINIMIZE_RATE_TARGET : y;});
```
Comment (voj) [inline src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_SubductionInversionTargetMFDs.java:88]: Is the old MINIMIZE_RATE_BELOW_MAG still used now? If not, you could delete it

File: src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_SubductionInversionRunner.java [hunk 4]
```diff
@@ -17,6 +17,9 @@
  */
 public class NZSHM22_SubductionInversionRunner extends NZSHM22_AbstractInversionRunner {
 
+	protected double mfdMinMag;
```
Comment (voj) [inline src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_SubductionInversionRunner.java:20]: you could shorten this to 
```
protected double mfdMinMag = 7.05;
```

**Review by voj — CHANGES_REQUESTED** [review]
Looks good! minor comments

**Review by chrisbc — APPROVED** [review]
@chrisdicaprio all good for me thanks - @voj has good advice :)
